system/View/View.php

```diff
@@ -134,6 +137,17 @@ class View implements RendererInterface
      */
     protected $sectionStack = [];
 
+    /**
+     * View Decorators are class methods that will be run in sequence to
+     * have a chance to alter the generated output just prior to caching
+     * the results.
+     *
+     * All classes must implement CodeIgniter\View\ViewDecorator
+     *
+     * @var array
+     */
+    public $decorators = [];
+
```

Comment:
Ah, I think you actually meant this for `CodeIgniter\Config\View`. If you move this over to the correct class it will satisfy my comments about the Config files above.